src/main/webapp/css/scheduleavailability.css

```
@@ -0,0 +1,221 @@
+    #availabilityGrid thead th,
+    #availabilityGrid .availability-date {
+        position: static;
+    }
+
+    #availabilityGrid thead {
+        display: table-header-group;
+    }
+
+    #availabilityGrid tbody tr {
+        break-inside: avoid;
+        page-break-inside: avoid;
```

Stylelint (17.14.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)